### ISSUE TITLE
TimeoutOverflowWarning

### ISSUE THREAD

**Issue by matthieuh** [quirrel-dev/quirrel#190]
## Bug Report

**Current Behavior**
`TimeoutOverflowWarning` enqueuing in Blitz: `TimeoutOverflowWarning: 2650483302 does not fit into a 32-bit signed integer.`

**Input Code**

```ts
await reminderQueue.enqueue({
    to: "Sandy Cheeks",
    message: "Howdy!",
  }, {
    repeat: {
      cron: '0 10 18 * *'
    }
  })
```

**Expected behavior/code**
No warnings

**Environment**

- Quirrel version: v1.3.1
- Node/npm version: v12.13.1

**Additional context/Screenshots**
<img width="689" alt="Screen Shot 2021-03-18 at 18 48 52" src="https://user-images.githubusercontent.com/1550192/111672810-9e391380-881a-11eb-976c-0cacaac0d611.png">

**Comment by Skn0tt** [comment]
Thanks for the report! Working on it ☺️

**Comment by Skn0tt** [comment]
@allcontributors add matthieuh for bugs

**Comment by Skn0tt** [comment]
published in v1.3.2
